unit-test.php

```diff
@@ -0,0 +1,107 @@
+<?php
```

Comment:
I assume this file doesn't needed anymore?

Reply:
yes, you are right